packages/core/solidity/src/account.ts

```diff
@@ -175,6 +176,23 @@ function addERC7579Modules(c: ContractBuilder, opts: AccountOptions): void {
   );
 }
 
+function addMultisigFunctions(c: ContractBuilder, opts: AccountOptions): void {
+  switch (opts.signer) {
+    case 'MultisigWeighted':
+      c.addFunctionCode(
+        `_setSignerWeights(${functions.setSignerWeights.args.map(({ name }) => name).join(', ')});`,
+        functions.setSignerWeights,
+      );
+    // eslint-disable-next-line no-fallthrough
+    case 'Multisig':
+      c.addFunctionCode(`_addSigners(${functions.addSigners.args[0]?.name});`, functions.addSigners);
+      c.addFunctionCode(`_removeSigners(${functions.removeSigners.args[0]?.name});`, functions.removeSigners);
+      c.addFunctionCode(`_setThreshold(${functions.setThreshold.args[0]?.name});`, functions.setThreshold);
```

Comment:
Can these use non-null assertions since we know that args[0] is available?
```suggestion
      c.addFunctionCode(`_addSigners(${functions.addSigners.args[0]!.name});`, functions.addSigners);
      c.addFunctionCode(`_removeSigners(${functions.removeSigners.args[0]!.name});`, functions.removeSigners);
      c.addFunctionCode(`_setThreshold(${functions.setThreshold.args[0]!.name});`, functions.setThreshold);
```